### DISCUSSION TITLE
(use case) Links for receive donations on the Block Hub?

### DISCUSSION THREAD

**Discussion by ghost** [blockprotocol/blockprotocol#639] [Q&A] [answered]
Hi everyone.

#### Idea
I would like to know if it would be possible to receive donations in the block hub to encourage the use of the block protocol. 

#### concept
![image](https://user-images.githubusercontent.com/111701513/207516852-af611eb4-20bd-4895-8ada-ae8a27a11817.png)

**image-description:** As we can see in the image we can send or receive some financial donation to create, maintain and update components in Block Hub.

#### another concept
![image](https://user-images.githubusercontent.com/111701513/207530607-9da72315-d657-4751-9fd2-8fe5500f1148.png)

**image-description:** As we can see in the image we can send or receive some financial donation to create, maintain and update components in Block Hub too.

#### feedback/question
*Would it be possible to receive donations in the block hub to encourage the use of the block protocol?*

**Comment by vilkinsons** [comment, marked as answer]
Hi @codehangen, definitely. I think we could adopt something like the `FUNDING.yml` format used by GitHub (now even used by ourselves in [this very repo](https://github.com/blockprotocol/blockprotocol/blob/main/.github/FUNDING.yml). Initially I don't think we'd want to be involved in processing these ourselves for reasons of overhead and complexity, but we could certainly link out to a handful of approved/trustworthy external services that offer this (including, I'd imagine, GitHub).

Perhaps if a community developer feels strongly about this they can chime in and propose an approach. The site's codebase is public in this repository, so anybody would be free to work on it.

**Reply by ghost** [reply]
Hi nonparibus.

> Hi @codehangen, definitely. I think we could adopt something like the FUNDING.yml format used by GitHub (now even used by ourselves in [this very repo](https://github.com/blockprotocol/blockprotocol/blob/main/.github/FUNDING.yml). Initially I don't think we'd want to be involved in processing these ourselves for reasons of overhead and complexity, but we could certainly link out to a handful of approved/trustworthy external services that offer this (including, I'd imagine, GitHub).
> Perhaps if a community developer feels strongly about this they can chime in and propose an approach. The site's codebase is public in this repository, so anybody would be free to work on it.

thank you for feedback again.
